src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -12,6 +13,37 @@
 from ..services.auth import JwtManager
 
 
+# minimum password complexity rules
+def check_password_complexity(password, username):
+    """
+    Suggested rules for password complexity
+    - at least 6 chars in length
+    - must not be the same as the user account name
+    - consist of characters from the following groups
+    - alphabetic a-z, A-Z
```

Comment:
This will lock out every language (e.g. chinese, ...) except those that are based on latin alphabet.